### DISCUSSION TITLE
Onboarding Commit message by Renovate does not follow conventional commit

### DISCUSSION THREAD

**Discussion by Taiki0807** [renovatebot/renovate#21616] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

## GitHub Actions not passed
[https://github.com/Taiki0807/next.js-template/actions/runs/4745704765](https://github.com/Taiki0807/next.js-template/actions/runs/4745704765)

I would like to resolve this so that it can pass.

### What you tried so far.

I searched but could not find a solution.

**Comment by MindTooth** [comment]
You might need to add:

```json
{
  "extends": [":semanticCommits"]
}
```

https://docs.renovatebot.com/semantic-commits/#manually-enabling-or-disabling-semantic-commits

**Reply by HonkingGoose** [reply]
I think the user's question is really: "How do I get Renovate to use conventional commits for it's onboarding PR?" The GitHub Action run the user linked to is for this onboarding PR:

- https://github.com/Taiki0807/next.js-template/pull/1

They haven't merged that onboarding PR, so Renovate does not update any dependencies yet. Renovate sees that the repository uses conventional commits, but does not change the Onboarding PR's title to something like `chore: onboard Renovate`.

**Reply by Taiki0807** [reply]
``` json
{
  "extends": [":semanticCommits"]
}
```
I've made changes to this.

[https://github.com/Taiki0807/next.js-template/actions/runs/4753660624/jobs/8445503789?pr=1](https://github.com/Taiki0807/next.js-template/actions/runs/4753660624/jobs/8445503789?pr=1)

Is there any way to fix this?

**Reply by rarkins** [reply]
You can check the branch out locally and force-push a commit with conventional syntax

**Reply by Taiki0807** [reply]
Do I change the commit message in Add renovate.json?

**Reply by HonkingGoose** [reply]
Easiest fix:

1. Create new branch `chore/configure-renovate-manually`
1. Copy/paste onboarding `renovate.json` file into your branch
1. Open PR, let tests run
1. Merge PR, and enjoy Renovate 😄

**Reply by Taiki0807** [reply]
Thanks for the reply.
[https://github.com/Taiki0807/next.js-template/pull/1](https://github.com/Taiki0807/next.js-template/pull/1)
Is it okay to close this pull request?

**Reply by HonkingGoose** [reply]
Even better, once Renovate sees you have a `renovate.json` file on `main` it will auto-close the configuration PR. 😉

**Reply by Taiki0807** [reply]
OK
Thank you

**Reply by Taiki0807** [reply]
[https://github.com/Taiki0807/next.js-template/pull/2](https://github.com/Taiki0807/next.js-template/pull/2)
Will it work if I merge these?

**Reply by Taiki0807** [reply]
[https://github.com/Taiki0807/next.js-template](https://github.com/Taiki0807/next.js-template)
Merged. is renovate working properly?

**Reply by HonkingGoose** [reply]
Renovate is working, because you're now have a Dependency Dashboard issue. 😉 Renovate will start creating PRs within a few hours.

It takes time for Renovate to run, read [Renovate docs, Known limitations](https://docs.renovatebot.com/known-limitations/) to learn more.

**Reply by Taiki0807** [reply]
Ok
Do I have to set it up using the same method as I did this time, every time I use renovate?

**Reply by rarkins** [reply]
Yes, until the issue is fixed

**Reply by HonkingGoose** [reply]
We have an issue for Renovate to use semantic commits for the onboarding PR title:

- #21618

Until we fix that problem you'll have to make a PR that matches the Semantic Commits rules. The easiest way is to copy/paste the `renovate.json` config file into a new PR with the correct commit and PR title.

In repositories where you **don't** use Semantic Commits you can probably merge the Renovate Onboarding PR as-is, after checking the config of course.

I hope you enjoy Renovate bot! 😄

**Reply by Taiki0807** [reply]
Thank you for your support, I appreciate it.

**Comment by rarkins** [comment, marked as answer]
I think this used to work so we can classify it as a bug in an issue. It should be titled "Use semantic commits for onboarding PR title when detected"

**Reply by HonkingGoose** [reply]
- #21618
